<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortforPage</class>
 <widget class="QWidget" name="PortforPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background-color:#1c1d25;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Logolabel">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="connlabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>214</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="portlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>269</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dashlabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>326</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>211</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:#445163;
border:0.5px solid #000000;
}</string>
    </property>
    <property name="text">
     <string>             CONNECT TO SERVER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ToPort">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>256</y>
      <width>211</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:#ffb400;
border:0.5px solid #000000;
}</string>
    </property>
    <property name="text">
     <string>PORTFORWARD  </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>312</y>
      <width>211</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:#445163;
border:0.5px solid #000000;
}</string>
    </property>
    <property name="text">
     <string>   VPN DASHBOARD</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background-color:#1c1d25;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="facelabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>14</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="welclabel">
    <property name="geometry">
     <rect>
      <x>683</x>
      <y>13</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:#445163;

}</string>
    </property>
    <property name="text">
     <string>Welcome back Vecious</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>891</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background-color:#242533;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color:#18191e;
border:#18191e;
color:#979696;
border-radius:10px;
padding-left:12px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Port..</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color:#18191e;
border:#18191e;
color:#979696;
border-radius:10px;
padding-left:12px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Internal IP..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>370</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#ffb400;
border-radius:10px;
color:#445163;
}</string>
    </property>
    <property name="text">
     <string>Port Forward</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color:#262836;
color:#53e207;
padding: 0 0 30 0px;
border-radius:10px;

}</string>
    </property>
    <property name="text">
     <string>Portforwarded</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color:#ffffff;
background-color:#262836;
}</string>
    </property>
    <property name="text">
     <string>4392</string>
    </property>
   </widget>
   <widget class="QComboBox" name="locationcombo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
background-color:#18191e;
border-radius:10px;
color:#979696;
padding-left:12px;
text-color:white;
border:#18191e;
}
QComboBox:editable {
    background: red;
    color: pink;
}
QComboBox::drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 35px;
     color: #979696;
     border-left-width: 0px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 10px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
     padding-left: 12px;
 }

QComboBox::down-arrow 
{image: url(:/resources/img/combo2.PNG);
 padding-right:25px;
}</string>
    </property>
    <property name="currentText">
     <string>Location</string>
    </property>
    <item>
     <property name="text">
      <string>Location</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>iouiouio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yio</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="locationtxt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="porttxt">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>230</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iptxt">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>300</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="portcombo">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>94</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
background-color:#242533;
border-radius:15px;
color:#f0f0f1;
border:#242533;
border:0.8px solid #f0f0f1;
padding-left:30px;

}
QComboBox::drop-down {border-width: 0px;}
QComboBox::down-arrow 
{image: url(:/resources/img/combo1.PNG);
 padding-right:20px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Open Port</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>Open Port</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
